src/App.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable no-nested-ternary */
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import AllProducts from './pages/AllProducts';
+import AddProduct from './pages/AddProduct';
+import MyReservations from './pages/MyReservations';
+import ProductDetails from './pages/ProductDetails';
+import Reserve from './pages/Reserve';
+import Sidebar from './components/Sidebar';
+import SigninForm from './components/SigninForm';
+import SignupForm from './components/SignupForm';
+import LoadingPage from './pages/LoadingPage';
+import Page404 from './pages/Page404';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/208409159-fe4a3fd1-b38e-48cc-a33f-3929aecd469f.png)